src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java

```diff
@@ -162,15 +162,22 @@ public final class SiteUtil {
      */
     private static final String V824 = "8.24";
 
+    /**
+     * Frequent version.
+     */
+    private static final String V50 = "5.0";
+
     /**
      * Map of properties whose since version is different from module version but
      * are not specified in code because they are inherited from their super class(es).
+     * Until <a href="https://github.com/checkstyle/checkstyle/issues/14052">#14052</a>.
      */
     private static final Map<String, String> SINCE_VERSION_FOR_INHERITED_PROPERTY = Map.ofEntries(
         Map.entry("MissingDeprecatedCheck.violateExecutionOnNonTightHtml", V824),
         Map.entry("NonEmptyAtclauseDescriptionCheck.violateExecutionOnNonTightHtml", "8.3"),
         Map.entry("NonEmptyAtclauseDescriptionCheck.javadocTokens", "7.3"),
-        Map.entry("FileTabCharacterCheck.fileExtensions", "5.0"),
+        Map.entry("FileTabCharacterCheck.fileExtensions", V50),
+        Map.entry("FileLengthCheck.fileExtensions", V50),
```

Comment:
fileExtensions was added to AbstractFileSetCheck at 3.0
https://github.com/checkstyle/checkstyle/commit/00fb7f0c1852eb015933c7c121284597de29caad#diff-61408aaabd0047105ba8afb9b532fd876d7bbf4f96802dab33ea6b3405d8221cR37

FileLengthCheck mostlikely from same 3.0 version.
https://github.com/checkstyle/checkstyle/commit/4b209708c6823dabedb3ba74bbfa92fe638f11be

our xdoc says 5.0 - https://checkstyle.org/checks/sizes/filelength.html#Properties
@rnveach, do you really want to change version in 3.0?

I doubt anybody care about such old versions.

Reply:
confirmed Check and property were appeared in 3.0

Reply:
> Check and property were appeared in 3.0

Are you saying our `since` version for `FileTabCharacterCheck` and `FileLengthCheck` is wrong as it says 5.0?
https://github.com/checkstyle/checkstyle/blob/5eb384ef18e5b3cf7c4f46e733437cf7e4df46d3/src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java#L74
https://github.com/checkstyle/checkstyle/blob/5eb384ef18e5b3cf7c4f46e733437cf7e4df46d3/src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/FileLengthCheck.java#L61

Reply:
I put 5.0 as you wanted.

Reply:
`AbstractFileSetCheck` is missing `since` version for `fileExtensions`, or is this in another PR?

Reply:
most likely, I update only what is required for current PR.